tests/points/api_tests.py

```diff
@@ -370,15 +370,15 @@ def test_timeseries_with_multiple_datasets_but_one_is_bad(self):
 
     def test_timeseries_with_a_tree_filter(self):
         endpoint = 'timeseries'
-        query = '?obs_date__ge=2005&agg=year'
+        query = '?obs_date__ge=2005-01-01&agg=year'
```

Comment:
So it turns out that if you just provide a date argument of 2005, dateutil.parser will convert that the date in 2005 that corresponds to the day you make the query.

For example, today is the 28th of July, and so 2005 becomes 2005-07-28.

Since the date doesn't stay the same, the test ends up failing because of the different results that could be produced depending on the day you ran that test.